terra-app-setup-chart/values.yaml

```diff
@@ -6,7 +6,7 @@ serviceAccount:
     gcpServiceAccount:
 
 relaylistener:
-  image: terradevacrpublic.azurecr.io/terra-azure-relay-listeners:3a932af
+  image: terradevacrpublic.azurecr.io/terra-azure-relay-listeners:decc8b0
```

Comment:
This is overridden by Leo conf anyway ([link](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/reference.conf#L210), just updating helm to match it.